### DISCUSSION TITLE
Mancanza della Notifica di Trascrizione per il Citt_046 (Trascrizione del decreto per matrimonio-unione civile-naturalizzazione-da consolato).

### DISCUSSION THREAD

**Discussion by gcuci** [italia/ansc#1061] [Miglioramenti]
Buongiorno,

per il caso d'uso in oggetto, non viene generata una Notifica di Trascrizione interna ad ANSC, e conseguentemente, neanche una notifica in ANPR per il Comune. Questo, probabilmente, perché il cittadino ha la residenza all'estero.

In questo caso, però, alla validazione e firma dell'evento, segue l'apertura in Anagrafe di una pratica di iscrizione all'AIRE, regolarmente supportata dalla documentazione consolare per l’iscrizione, salvo rari casi di mancata iscrizione.

Riteniamo quindi, che anche per questo caso d'uso, nonostante la mancanza della residenza, debbano essere comunque generate le Notifiche.

In attesa di riscontro,

cordiali saluti.

**Comment by asqueglia** [comment]
Buon pomeriggio, utilizzando il caso d'uso indicato se viene specificato il comune AIRE del soggetto intestatario viene generata la notifica verso l'anagrafe; l'ufficiale di anagrafe deciderà poi di procedere con l'eventuale iscrizione anagrafica. Chiudiamo pertanto la discussion
